packages/component-library/stories/CheckboxGroup.stories.tsx

```diff
@@ -42,6 +43,47 @@ export default class CheckboxGroupExample extends React.Component<Props> {
   }
 }
 storiesOf("CheckboxGroup (React)", module)
+  .addDecorator(withKnobs)
+  .add("with Knobs", () => {
+    return (
+      <CheckboxGroup labelText={text("labelText", "Checkbox Group Label")}>
+        <CheckboxGroupExample
+          render={({ checkedStatus, onCheckHandler }) => (
+            <CheckboxField
+              onCheck={onCheckHandler}
+              id="checkbox-1"
+              checkedStatus={checkedStatus as any}
```

Comment:
nitpicking, but could this be cast to a string instead? That will bring it inline with the typescript definition on `CheckboxField`

Reply:
I have to admit I copied and pasted the earlier example, but yeah, that could be fixed.